### PR TITLE
Fix Resolve Expression Attribute Candidates With Qualifier

### PR DESCRIPTION
**Pull request by shehabgamin**

### CODE REVIEW

File: crates/sail-plan/src/resolver/expression.rs [hunk 1]
```diff
@@ -673,6 +673,13 @@ impl PlanResolver<'_> {
     ) -> PlanResult<Vec<(String, DataType, Column)>> {
         // TODO: handle qualifier and nested fields
         let name: Vec<String> = name.clone().into();
+        let name = if name.iter().any(|ident| ident.contains('.')) {
```
Comment (linhr) [inline crates/sail-plan/src/resolver/expression.rs]: The identifier is considered "raw". If the nested identifier is not parsed correctly, it should be fixed upstream in the protobuf or SQL processor.

Comment (shehabgamin) [reply crates/sail-plan/src/resolver/expression.rs]: This is coming from proto. Fixing it upstream would require doing it everywhere in proto? Or is there somewhere common to place this?

Comment (linhr) [reply crates/sail-plan/src/resolver/expression.rs]: Let’s see if fixing this FIXME would fix the issue.

<https://github.com/lakehq/sail/blob/fca61be759ea02071b9f290b3969e6f2a9ccb3a6/crates/sail-spark-connect/src/proto/expression.rs#L38-L39>

Comment (shehabgamin) [reply crates/sail-plan/src/resolver/expression.rs]: worked!

**Review by linhr — APPROVED** [review]

**Pull request merged**